x/poe/types/msg.go

```diff
@@ -80,8 +82,14 @@ func (msg MsgCreateValidator) ValidateBasic() error {
 		return stakingtypes.ErrEmptyValidatorPubKey
 	}
 
-	if !msg.Value.IsValid() || !msg.Value.Amount.IsPositive() {
-		return sdkerrors.ErrInvalidRequest.Wrap("delegation amount")
+	if msg.Amount.IsNil() || !msg.Amount.IsValid() {
```

Comment:
Technically yes for the internal status of sdk.Int. 
Our CLI command would fail but people may use a different client so that we need to double check on the server side as well.